app/models/content_store_payload_version.rb

```diff
@@ -0,0 +1,41 @@
+class ContentStorePayloadVersion < ActiveRecord::Base
+  def self.increment(content_item_id:)
```

Comment:
I think this could be a regular argument, defaulting to `nil`, instead of a keyword argument. This means callers wouldn't have to pass a named argument, given the method signature only has an (assumed, given they're keyword arguments) arity of one.